### PR TITLE
validate whether wallet activation data is equal to from or to tx hash

### PR DESCRIPTION
**Pull request by josipkoncurat**

### CODE REVIEW

**Comment by josipkoncurat** [comment]
Just a quick dirty fix. Changes are needed on blochchain.

**Comment by githinho** [comment]
What does this fixes? Only transactions made by admin?

**Comment by filipdujmusic** [comment]
> What does this fixes? Only transactions made by admin?

Yes. For now. When coop_id gets integrated in blockchain service, all accounts are going to have tx_hash data (including admin) and then we'll standardize these calls

File: src/main/kotlin/com/ampnet/reportservice/service/impl/TemplateDataServiceImpl.kt [hunk 1]
```diff
@@ -146,10 +146,11 @@ class TemplateDataServiceImpl(
 
     private fun validateTransactionBelongsToUser(userWallet: WalletResponse, fromTxHash: String, toTxHash: String) {
         val txHash = userWallet.hash
-        if (txHash != fromTxHash && txHash != toTxHash) {
+        val activationData = userWallet.activationData
```
Comment (githinho) [inline src/main/kotlin/com/ampnet/reportservice/service/impl/TemplateDataServiceImpl.kt:149]: Add a comment in code explaining this problem:
https://github.com/AMPnet/report-service/pull/24#issuecomment-731120648

**Review by githinho — CHANGES_REQUESTED** [review]

**Review by githinho — APPROVED** [review]

**Pull request merged**